app.js

```diff
@@ -38,26 +37,51 @@ var app = express();
 var server = https.createServer(options, app);
 var expressWs = expressWs(app, server);
 
-// Command line arguments
-var getopt = new GetOpt([
-  ['d', 'debug',      'Enable debug features'],
-  ['p', 'port=PORT',  'Specify the server port to use (default ' + port + ')'],
-  ['h', 'help',       'Display help' ],
-  ['v', 'verbose',    'Show verbose output' ],
-]);
+function configureOptions() {
+  if (!config.get('cli')) {
+    return {
+      options: {
+        debug: false,
+        port: null,
+      }
+    };
+  }
+
+  // Command line arguments
+  const getopt = new GetOpt([
+    ['d', 'debug', 'Enable debug features'],
+    ['p', 'port=PORT', 'Specify the server port to use (default ' + port + ')'],
+    ['h', 'help', 'Display help' ],
+    ['v', 'verbose', 'Show verbose output'],
+  ]);
+
+  const opt = getopt.parseSystem();
+  const options = {
+    debug: !!opt.options.debug, // cast to bool
+    verbose: opt.options.verbose,
+  }
 
-var opt = getopt.parseSystem();
-if (opt.options.verbose) {
+  if (opt.options.verbose) {
     console.log(opt);
-}
+  }
 
-if (opt.options.help) {
+  if (opt.options.help) {
     getopt.showHelp();
     process.exit(1);
+  }
+
+  if (opt.options.port) {
+    options.port = parseInt(opt.options.port);
+  }
+
+  return {
+    options
+  };
 }
 
-if (opt.options.port) {
-  port = parseInt(opt.options.port);
+const opt = configureOptions();
+if (opt.port) {
```

Comment:
Either this should be `opt.options.port`, or `configureOptions` should return the `options` object directly instead of `{ options }`